src/syscall/syscall_libc_wasi.go

```diff
@@ -324,8 +324,7 @@ func Pipe2(p []int, flags int) (err error) {
 }
 
 func Getpagesize() int {
-	// per upstream
-	return 65536
+	return libc_getpagesize()
 }
```

Comment:
Probably unnecessary, the WebAssembly page size is fixed at 64kB. I don't really see a benefit of calling a libc function.
On the other hand, it probably won't increase binary size by much either so :shrug:

Reply:
Mostly for consistency, and to limit hard-coded numbers. I have no idea whether wasi will evolve in some direction as to not be 64KiB, but if it does, then that information will be encoded in the SDK instead of here.